### PR TITLE
Show withdrawn bids in separate table and hide my bids if there are none

### PR DESCRIPTION
**Pull request by Mvii7**

### CODE REVIEW

File: src/app/auction-page/auction-page.component.html [hunk 1]
```diff
@@ -324,6 +312,64 @@
   </div>
 </section>
 
+<section class="axn-section" *ngIf="account && showAuctions && hasWithdrawnBids">
+  <div class="axn-form-block p-20">
+    <div class="axn-title-block p-20">
+      <span class="axn-title">Withdrawn Bids</span>
```
Comment (Mvii7) [inline src/app/auction-page/auction-page.component.html:318]: Alternative is to call this Bid History

Comment (Vicarious1) [reply src/app/auction-page/auction-page.component.html:318]: I like Bid History more than Withdrawn Bids, but it's not a big deal if you want to leave it this way

**Review by Vicarious1 — APPROVED** [review]
LGTM

File: src/app/auction-page/auction-page.component.html (hunk 1 above)
Comment (AudioSin) [reply src/app/auction-page/auction-page.component.html:318]: I prefer Bid History as well. Also I think it makes more sense to place the active auctions first and after that the history no?

Comment (Mvii7) [reply src/app/auction-page/auction-page.component.html:318]: That's how it's placed now

**Pull request merged**